dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/runner/task/CommonTaskProcessor.java

```diff
@@ -47,6 +52,7 @@ public class CommonTaskProcessor extends BaseTaskProcessor {
 
     @Override
     protected boolean submitTask() {
+        checkAndReplaceTestDataSource();
```

Comment:
Optimize the function name, convert the online environment to the test environment.